__tests__/app.test.js

```diff
@@ -149,6 +149,78 @@ describe("GET 200: /api/articles", () =>{
   
 })
 
+describe("POST 201: /api/articles/:article_id/comments", ()=>{
+  test("sends a 201 status and returns the posted comment after successfully inserting comment", () =>{
+    const comment = {username: "butter_bridge", body:"this is a comment"}
+    
+    return request(app).post("/api/articles/4/comments").send(comment).expect(201).then((response) =>{
+      const comments = response.body
+      expect(comments[0]).toHaveProperty("comment_id", expect.any(Number));
+        expect(comments[0]).toHaveProperty("votes", expect.any(Number));
+        expect(comments[0]).toHaveProperty("author", expect.any(String));
```

Comment:
We shouldn't be checking for 'any' in any keys where we know the value. You have any expect.any here and an expect toBe later down the file for specific strings ('this is a comment', 'butter_bridge'). Please remove these less specific statements